test/testcafe/spec/ChallengeOktaVerifyFastPassView_spec.js

```diff
@@ -201,6 +279,50 @@ async function setupLoopbackFallback(t) {
   return deviceChallengeFalllbackPage;
 }
 
+test
+  .requestHooks(loopbackPollingLogger, loopbackRedundantPollingMock)('in loopback server, redundant polling exists if server returns enhancedPollingEnabled as false', async t => {
```

Comment:
Once you rebase the latest from master, you will want to disable this test for the third Gen SIW, please add this ticket in a comment above this test and make the below change to disable it from running against Gen3 🙏 
```suggestion
// TODO: OKTA-623228 - fix for Gen 3 SIW
test
  .meta('v3', false)
  .requestHooks(loopbackPollingLogger, loopbackRedundantPollingMock)('in loopback server, redundant polling exists if server returns enhancedPollingEnabled as false', async t => {
```